aggregator/src/aggregator/http_handlers.rs

```diff
@@ -128,7 +125,12 @@ async fn run_error_handler(error: &Error, mut conn: Conn) -> Conn {
         Error::UnrecognizedCollectionJob(_, _) => conn.with_status(Status::NotFound),
 
         Error::UnauthorizedRequest(task_id) => conn.with_problem_document(
-            &ProblemDocument::new_dap(DapProblemType::UnauthorizedRequest).with_task_id(task_id),
+            &ProblemDocument::new(
+                "https://docs.divviup.org/references/janus-errors#unauthorized-request",
+                "The request's authorization is not valid.",
+                Status::Forbidden,
+            )
+            .with_task_id(task_id),
```

Comment:
I think just using a 403 status code would be sufficient for this case (similar to the UnrecognizedCollectionJob case above). If we want to provide more detail, we could still use a problem document with `about:blank`.